### PR TITLE
Rename dimensions and segments with Server time to Site Time/Time in UTC

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Started renaming dimensions and segments starting with name Server time to Site Time/Time in UTC

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
@AltamashShaikh Is there a special reason why you renamed some of the segments?
Changing names of segments is a bit critical. Anyone using that segment might afterwards see errors as the segment gets invalid.

**Comment by AltamashShaikh** [comment]
> @AltamashShaikh Is there a special reason why you renamed some of the segments? Changing names of segments is a bit critical. Anyone using that segment might afterwards see errors as the segment gets invalid.

@sgiehl My bad, updating `$nameSingular` should be sufficient, I read the comments in tickets incorrectly and updated the segment too

**Comment by sgiehl** [comment]
@AltamashShaikh no problem. Guess you can now also remove all the expected test files you had newly added.

File: plugins/VisitTime/lang/en.json [hunk 1]
```diff
@@ -5,32 +5,49 @@
...
         "ColumnVisitEndServerDate": "Server time - date (Time of last action)",
+        "ColumnVisitEndSiteDate": "Time in UTC - date (Time of last action)",
         "ColumnVisitEndServerDayOfMonth": "Server time - day of month (Time of last action)",
+        "ColumnVisitEndSiteDayOfMonth": "Time in UTC - day of month (Time of last action)",
         "ColumnVisitEndServerDayOfWeek": "Server time - day of week (Time of last action)",
+        "ColumnVisitEndSiteDayOfWeek": "Time in UTC - day of week (Time of last action)",
         "ColumnVisitEndServerDayOfYear": "Server time - day of year (Time of last action)",
+        "ColumnVisitEndSiteDayOfYear": "Time in UTC - day of year (Time of last action)",
         "ColumnVisitEndServerQuarter": "Server time - quarter (Time of last action)",
+        "ColumnVisitEndSiteQuarter": "Time in UTC - quarter (Time of last action)",
```
Comment (sgiehl) [inline plugins/VisitTime/lang/en.json]: It's actually a bit confusing that all translations keys are named `Site`, even though most of them actually don't contain a site related time, but UTC. Maybe renaming the keys where UTC is used would make it a bit less confusing when looking through the changes.

Comment (AltamashShaikh) [reply plugins/VisitTime/lang/en.json]: @sgiehl updated

**Review by sgiehl — APPROVED** [review]

**Pull request merged**